### DISCUSSION TITLE
crazyflie2.1 can't reboot can't shutdown; only M2 M3 blinks; unexpected termination of Update Crazyflie and AIdeck firmware program.

### DISCUSSION THREAD

**Discussion by chaoyanren** [bitcraze/discussions#1214] [Q&A] [answered]
I accidentally terminated the Update Crazyflie and AIdeck firmware program, and now crazyflie2.1 won't shut down (unless unplugged) and reboot; and now the M2 and M3 are blinking about 1Hz; and the client won't scan crazyfllie2.1!

Please help me.

Best regards

**Comment by chaoyanren** [comment]
![image](https://github.com/bitcraze/discussions/assets/42300897/51fd0e8d-2abc-4f0b-a4b6-82cdbcf55d33)
It can scan; but can not be conected.

**Comment by ataffanel** [comment, marked as answer]
If both blue LEDs blink blue, this means you are currently in bootloader mode which is a good thing! This is the channel 0 your client is showing.

In the client, you need to connect your Crazyflie bootloader in "cold boot", you will then be able to flash the full nrf and stm32 firmware. To update the aideck you can flash again by connecting the Crazyflie normally, this will then also update the decks.

**Comment by juhyun9** [comment]
You have to push the 'disconnect' or 'cancel(when your CF trying to connect to client)' button.
![image](https://github.com/user-attachments/assets/944cf106-1706-43a8-98b5-4dde286ecb1a)

and then click the 'initiate bootloader cold boot' in the cod boot (recovery section).
![image](https://github.com/user-attachments/assets/58900091-b6bb-4b61-b0a4-7485f669fbc6)

I solve same problem in this way
